libmamba/src/solver/libsolv/helpers.cpp

```diff
@@ -140,6 +147,15 @@ namespace mamba::solver::libsolv
             return util::lstrip_if_parts(tail, [&](char c) { return !is_sep(c); });
         }
 
+        void
+        get_fake_signatures(simdjson::dom::parser& fake_parser, simdjson::dom::object& fake_signatures)
+        {
+            // Getting a fake simdjson::dom::object to simulate empty signatures
+            // A valid empty simdjson::dom::object is not handled in simdjson
+            const auto fake_signatures_json = R"(  { "fake_signatures": "val" }  )"_padded;
+            fake_signatures = fake_parser.parse(fake_signatures_json).get_object().value();
+        }
+
```

Comment:
Why fake? Could empty be a better alternative instead instead?

Reply:
Hum because it's not really an empty json?